pkg/gateway/mirror/usecase/vald.go

```diff
@@ -69,23 +70,23 @@ func New(cfg *config.Data) (r runner.Runner, err error) {
 	// skipcq: CRT-D0001
 	cOpts = append(cOpts, grpc.WithErrGroup(eg))
 
-	client, err := mirror.New(
-		mirror.WithAddrs(cfg.Mirror.Client.Addrs...),
-		mirror.WithClient(grpc.New(cOpts...)),
+	mirrClient, err := client.New(
```

Comment:
I think it would be better to nameming as `mirrorClient` or `mClient`

Reply:
Thank you for the suggestion 🙏 
I apply it!